.gitignore

```
@@ -24,6 +24,8 @@ hs_err_pid*
 
 target/*
 
-.idea/workspace.xml
+.idea/.
```

`git rm ./idea -r` should clear up all these files

```suggestion
.idea/*
```